src/resources/postcss/base/forms/_text.pcss

```diff
@@ -24,9 +24,22 @@
 			padding: 12px 18px;
 			width: 100%;
 
+			body#top & { /* Enfold compatibility */
+				@mixin desktop-body-2-bold;
```

Comment:
same question here, are all styles affected or can we target only those affected?

Reply:
Specifically for this one, yes. They're setting styles for the input font, font weight and font color, padding and borders.

Reply:
great, sounds good